app/forms/journeys/early_years_payment/provider/authenticated/paye_reference_form.rb

```diff
@@ -0,0 +1,31 @@
+module Journeys
+  module EarlyYearsPayment
+    module Provider
+      module Authenticated
+        class PayeReferenceForm < Form
+          attribute :paye_reference
+
+          PAYE_REFERENCE_REGEX_FILTER = /\A\d{3}\/([A-Z]|\d){1,10}\z/
+
+          validates :paye_reference, presence: {message: i18n_error_message(:valid)},
+            format: {with: PAYE_REFERENCE_REGEX_FILTER, message: i18n_error_message(:valid)}
+
+          attr_reader :nursery_name
+
+          def initialize(journey_session:, journey:, params:)
+            super
+
+            @nursery_name = EligibleEyProvider.find_by(urn: answers.nursery_urn)&.nursery_name
+          end
```

Comment:
not easier with just a getter method?
```suggestion
          def nursery_name
            @nursery_name ||= EligibleEyProvider.find_by(urn: answers.nursery_urn)&.nursery_name
          end
```